Playgrounds/Homework3.playground/Contents.swift

```diff
@@ -0,0 +1,21 @@
+// Homework 3
+
+var svc = SomeAnalyticsService()
```

Comment:
I would suggest better naming here. Swift is talkative and usually self describing by naming.

Reply:
Do you mean for the `svc` variable?